token-lending/program/tests/borrow_obligation_liquidity.rs

```diff
@@ -408,3 +411,351 @@ async fn test_borrow_too_large() {
         )
     );
 }
+
+#[tokio::test]
+async fn test_borrow_max_receive_less_than_slippage() {
+    let mut test = ProgramTest::new(
+        "spl_token_lending",
+        spl_token_lending::id(),
+        processor!(process_instruction),
+    );
+
+    // limit to track compute unit increase
+    test.set_compute_max_units(60_000);
+
+    const FEE_AMOUNT: u64 = 5000;
+    const HOST_FEE_AMOUNT: u64 = 1000;
+
+    const USDC_DEPOSIT_AMOUNT_FRACTIONAL: u64 =
+        2_000 * FRACTIONAL_TO_USDC * INITIAL_COLLATERAL_RATIO;
+    const SOL_BORROW_AMOUNT_LAMPORTS: u64 = 50 * LAMPORTS_TO_SOL;
+    const USDC_RESERVE_COLLATERAL_FRACTIONAL: u64 = 2 * USDC_DEPOSIT_AMOUNT_FRACTIONAL;
+    const SOL_RESERVE_LIQUIDITY_LAMPORTS: u64 = 2 * SOL_BORROW_AMOUNT_LAMPORTS;
+    const SLIPPAGE_LIMIT: u64 = SOL_BORROW_AMOUNT_LAMPORTS - FEE_AMOUNT - 1;
```

Comment:
How about making this exactly at the limit to test your `<` condition, so instead this becomes:
```suggestion
    const SLIPPAGE_LIMIT: u64 = SOL_BORROW_AMOUNT_LAMPORTS - FEE_AMOUNT;
```
Then the test name would be `test_borrow_max_receive_minimum` or something

Reply:
Sorry, it looks like this one got missed -- is it possible for this to be the exact limit?

Reply:
Whoops, sorry! 😅